x-pack/heartbeat/monitors/browser/synthexec/enrich.go

```diff
@@ -207,6 +211,11 @@ func (je *journeyEnricher) createSummary(event *beat.Event) error {
 		},
 	})
 
+	_, err := event.GetValue("monitor.id")
```

Comment:
Do we create summary events when there is no monitor.id field?

Reply:
I don't think so because if the field does not exist `GetValue` will error out as it does when we try to extract fields within the logger itself.